### DISCUSSION TITLE
Curso FullStack - Padronizando endpoints

### DISCUSSION THREAD

**Discussion by danilobicas-unoeste** [balta-io/forum#604] [Tira Dúvidas] [answered]
Olá.
Eu posso usar o "modelo antigo" configurando os controllers mesmo? Ou no aspnet core precisa usar da forma que você fez? quais os pros e contras do modelo comum (CategoriesController, etc)? E como ficaria a chamado dos request e handlers no modelo comum???

Obrigado

Atte.
Danilo

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @danilobicas-unoeste, como vai?

Pode sim, tranquilamente, a diferença é que o Minimal APIs é mais enxuto, não leva toda arquitetura e pipeline que o MVC leva.
Mas se você acredita ser mais organizado com o MVC, segue em frente.

Só recomendo não deixar o Minimal API de lado, faz uma versão com ele também, mesmo que seja só para estudos!
